src/index.js

```diff
@@ -1,4 +1,4 @@
-/* global graphData countries countryPerformanceOnRiskViews asn tempData risks*/
+/* global graphData countries countryPerformanceOnRiskViews asn tempData risks ASPerformanceViews tempDataForASPerformance*/
 import React from 'react';
```

Comment:
Oh, actually I was talking about `tempData`.
I will remove `tempDataForASPerformance` as it was only for testing - now not in use.